ding2.profile

```diff
@@ -824,3 +791,35 @@ function ding2_set_cookie_page() {
   // display EU Cookie Compliance popup: anonymous user, authenticated user
   // administer EU Cookie Compliance popup: administrators, local administrator
 }
+
+/**
+ * Setup customerror pages for 403 and 404 status codes.
+ */
+function ding2_set_customerror_pages() {
```

Comment:
:+1: Great refactor here. This will also allow provide a foundation for a function that can be executed multiple times over the lifetime of a project if we want to change the default not found page.